apps/products-feed/src/modules/app-configuration/app-config.ts

```diff
@@ -98,37 +98,43 @@ export class AppConfig {
 
   setS3(s3Config: z.infer<typeof s3ConfigSchema>) {
     try {
+      logger.debug("Setting S3 config", { s3Config });
       this.rootData.s3 = s3ConfigSchema.parse(s3Config);
 
+      logger.info("S3 config saved");
       return this;
     } catch (e) {
-      logger.info(e, "Invalid S3 config provided");
+      logger.error(e, "Invalid S3 config provided");
```

Comment:
- Logger accepts `string` as first argument and payload in second argument
- To properly display error in Datadog, you need following:
```
logger.error('Message', { error: e })
```

so field `error` is parsed properly